students/eoner/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,11 @@
+from decimal import Decimal 
```

Comment:
I like how you explored using Decimal. Thanks.

Reply:
thanks - pretty crude implementation